src/index.html

```diff
@@ -16,7 +32,150 @@
       defer
     ></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <!-- #region header -->
+  <body class="page__body">
+    <header class="header">
+      <div class="top-bar">
+        <a
+          class="top-bar__logo"
+          href="#"
+        >
+          <img
+            src="./images/themet.png"
+            alt="Base logo"
+          />
+        </a>
+        <div class="top-bar__icons">
+          <a
+            href="#"
+            class="icon icons__icon--phone"
```

Comment:
Should show contact info on hover:
![image](https://github.com/mate-academy/layout_landing-page/assets/52002856/1001ba78-c954-417d-8543-4a744960e6bd)

Reply:
Done